lychee-bin/src/options.rs

```diff
@@ -179,6 +180,11 @@ pub(crate) struct Config {
     #[serde(default)]
     pub(crate) dump: bool,
 
+    /// Suggest link replacements for broken links, using a web archive. Only supports "wayback"
+    /// for the moment
+    #[arg(long, default_value = None)]
```

Comment:
Good point. Also made this change in https://github.com/thomas-zahner/lychee/pull/1.

Reply:
Thank you very much for the feedback the PR. I totally agree and have merged it.
I have update the flags like you suggested.
The default value is now "wayback" for the `--archive` flag.
Because we now use a value_parser for parsing the archive option, we must provide a string for `default_value`.

I've tried the following, but it doesn't work, because `clap::builder::OsStr` is expected:

`#[arg(long, value_parser = parse_archive_provider, default_value = Archive::default().to_string())]`

I couldn't figure out how to do it, so for now I just used `default_value = "wayback"`. Do you know how this could be done?